pkg/registry/servicecatalog/rest/storage_servicecatalog.go

```diff
@@ -133,6 +133,7 @@ func (p StorageProvider) v1alpha1Storage(
 				ResourceName:     tpr.ServiceClassKind.String(),
 				Separator:        "/",
 			},
+			HardDelete: true,
```

Comment:
>This simply makes all registry deletes of a ServiceClass a hard delete that will use the underlying storage interface Delete() func?

I don't understand the question. This only tells the storage interface's `Delete` function to make sure the object is removed after it returns.

>Am I right that this change is because there isn't actually any finalization that needs to be performed when deleting a ServiceClass? 

I have inferred this fact from the structure of `clientset_test.go`